docs/UserGuide.md

```diff
@@ -457,11 +462,8 @@ The `c` flag indicates that the task will be marked as completed, while the `u`
 ```
 > mark c 1
```

Comment:
Something seems to have gone wrong while automerging beyond this point. Automerge seems to have randomly duplicated or deleted certain sections and the indentation is a bit messed up also.

Reply:
This is resulted from version conflicts. Solved